.github/workflows/go-tests.yml

```
@@ -376,6 +376,7 @@ jobs:
       runs-on: ${{ needs.setup.outputs.compute-xl }}
       repository-name: ${{ github.repository }}
       go-tags: "${{ github.event.repository.name == 'consul-enterprise' && 'consulent consulprem consuldev' || '' }}"
+      go-version: "1.19"
```

I think we need to pin this to 1.19.10. The CI tests are currently passing, but I think that is only because the runner is using a [cached version of 1.19.10](https://github.com/hashicorp/consul/actions/runs/5546642282/jobs/10127253411). If/when the runner/action picks up 1.19.11 I expect the `go-test-api-1-19` tests to fail for the same reason.

```suggestion
      go-version: "1.19.10"
```